src/vcModals.cpp

```diff
@@ -1386,12 +1386,7 @@ void vcModals_DrawImageViewer(vcState *pProgramState)
       ImVec2 window = ImGui::GetWindowSize();
       ImVec2 windowPos = ImGui::GetWindowPos();
 
-#if GRAPHICS_API_OPENGL
-      ImVec2 uvs[2] = { {0,1}, {1,0} };
-#else
-      ImVec2 uvs[2] = { {0,0}, {1,1} };
-#endif
-      
+      ImVec2 uvs[2] = { {0,0}, {1,1} };    
```

Comment:
Does this still work for images that are loaded in?

Reply:
Yes.